static/js/addReviewClient.js

```diff
@@ -0,0 +1,49 @@
+const addReviewButton = document.querySelector(".add-review-button");
+let addReviewContent = document.querySelector(".add-review-content");
+
+addReviewButton.addEventListener("click", () => {
+  addReviewButton.classList.toggle("clicked");
+  addReviewContent.classList.toggle("show");
+});
+
+document
+  .querySelector(".submit-review")
+  .addEventListener("click", async (ev) => {
+    ev.preventDefault();
+    const movieId = window.location.href.split("/").pop();
+    const author = document.querySelectorAll("form > input")[0].value;
+    const rating = document.querySelectorAll("form > input")[1].value;
```

Comment:
These selectors are very generic. Imagine we'll add a modal with an input to subscribe to the newsletter to the page. It might also have a form with an input, and would break this code. I would change this to use classnames or IDs.

Reply:
You are right, and I already have IDs for those selectors. I dont know why I didnt use the ID:s. Thank you, and I will change it!